command/genesis/polybft_params.go

```diff
@@ -171,29 +167,14 @@ func (p *genesisParams) generateChainConfig(o command.OutputFormatter) error {
 		Bootnodes: p.bootnodes,
 	}
 
-	burnContractAddr := types.ZeroAddress
-
-	if p.isBurnContractEnabled() {
-		chainConfig.Params.BurnContract = make(map[uint64]types.Address, 1)
-
-		burnContractInfo, err := parseBurnContractInfo(p.burnContract)
-		if err != nil {
-			return err
-		}
-
-		if !p.nativeTokenConfig.IsMintable {
-			// burn contract can be specified on arbitrary address for non-mintable native tokens
-			burnContractAddr = burnContractInfo.Address
-			chainConfig.Params.BurnContract[burnContractInfo.BlockNumber] = burnContractAddr
-			chainConfig.Params.BurnContractDestinationAddress = burnContractInfo.DestinationAddress
-		} else {
-			// burnt funds are sent to zero address when dealing with mintable native tokens
-			chainConfig.Params.BurnContract[burnContractInfo.BlockNumber] = types.ZeroAddress
-		}
-	}
+	chainConfig.Params.BurnContract = make(map[uint64]types.Address, 1)
+	chainConfig.Params.BurnContract[0] = types.ZeroAddress
+	chainConfig.Genesis.BaseFee = p.parsedBaseFeeConfig.baseFee
+	chainConfig.Genesis.BaseFeeEM = p.parsedBaseFeeConfig.baseFeeEM
+	chainConfig.Genesis.BaseFeeChangeDenom = p.parsedBaseFeeConfig.baseFeeChangeDenom
```

Comment:
@goran-ethernal How we are going to decide should we enable the EIP-1559 or not? 
This configuration can and probably should be completely omitted if EIP-1559 is disabled.

Reply:
Let's do this through another PR. I agree, if `BaseFeeConfig` is not provided, we can consider that as a desire to disable EIP-1559 (London fork).